src/main/resources/templates/admin/logistic/in_processing.html

```
@@ -1,20 +1,167 @@
-<head>
+  <head>
     <div th:replace="~{fragments/head_info.html :: head_info}"></div>
-    <link rel="stylesheet" type="text/css" th:href="@{/css/profile.css}">
-    <link rel="stylesheet" type="text/css" th:href="@{/css/navbar.css}">
-    <link rel="stylesheet" type="text/css" th:href="@{/css/dashboard_table.css}">
-</head>
+    <link rel="stylesheet" type="text/css" th:href="@{/css/profile.css}" />
+    <link rel="stylesheet" type="text/css" th:href="@{/css/navbar.css}" />
+    <script src="/js/logistic.js" defer></script>
+    <script src="/js/bibliotecas/alpine.js" defer></script>
+  </head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)